src/rai_extensions/rai_nomad/README.md

```diff
@@ -0,0 +1,54 @@
+# RAI NoMaD
+
+This package provides a ROS2 Node which loads and runs the [NoMaD](https://general-navigation-models.github.io/nomad/index.html) model, that can be dynamically activated and deactivated using ROS2 messages.
+
+## Running instructions
+
+### 1. Download the .pth model weights file from [here](https://drive.google.com/file/d/1YJhkkMJAYOiKNyCaelbS_alpUpAJsOUb)
+
+### 2. Setup the ROS2 workspace
+
+In the base directory of the `RAI` package install dependancies:
+
+```
+poetry install --with nomad
+```
+
+Source the ros installation
+
+```
+source /opt/ros/${ROS_DISTRO}/setup.bash
+```
+
+Run the build process:
+
+```
+colcon build
+```
+
+Setup your shell:
+
+```
+source ./setup_shell.sh
+```
+
+### 3. Run the node
+
+Run the ROS2 node using `ros2 run`:
+
+```
+ros2 run rai_nomad nomad --ros-args -p model_path:="<path_to_weights>" -p image_topic:=<your_image_topic>
+```
```

Comment:
Could you please add default parameter for model_path? I think that by default the weights should be downloaded automatically.

Reply:
Done.